Sources/Player/Player.swift

```diff
@@ -186,25 +186,25 @@ public extension Player {
 }
 
 public extension Player {
-    /// Return whether the current player item player can be returned to live conditions.
-    /// - Returns: `true` if skipping to live conditions is possible.
-    func canSkipToLive() -> Bool {
-        guard timeRange.isValidAndNotEmpty, itemDuration.isIndefinite, chunkDuration.isValid else { return false }
-        return time < timeRange.end - chunkDuration
+    /// Return whether the current player item player can be returned to default position thanks to the stream type.
+    /// - Returns: `true` if skipping to default position is possible.
+    func canSkipToDefault() -> Bool {
+        switch streamType {
+        case .onDemand:
+            return true
+        case .dvr where chunkDuration.isValid:
+            return time < timeRange.end - chunkDuration
+        default:
+            return false
```

Comment:
We should discuss the behavior for livestreams:

1. Either we think skipping to default should be possible (to return to live conditions if the stream was paused and buffering was possible). In this case the `canSkipToDefault()` method should always return `true` for livestreams.
2. Or we think this should not be the case and the `skipToDefault` method should rather be:
    ```swift
    func skipToDefault(completion: @escaping (Bool) -> Void = { _ in }) {
        switch streamType {
        case .onDemand:
            queuePlayer.seek(to: .zero) { finished in
                completion(finished)
            }
        case .dvr:
            queuePlayer.seek(to: timeRange.end.clamped(to: timeRange)) { finished in
                completion(finished)
            }
        default:
            completion(true)
        }
    }
    ```